config/deploy/local/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+# Local deployment overlay.
+#
+# This requires the following .env files to exist in the project directory before
+# it can be used:
+# controller-config.env
+# aws-credentials.env
+
+resources:
+  - ../../default
+
+namespace: multi-cluster-traffic-controller-system
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+  - name: controller-config
+    envs:
+      - ../../../controller-config.env
+
+secretGenerator:
+  - name: aws-credentials
+    envs:
+      - ../../../aws-credentials.env
```

Comment:
I think in kcp-glbc we just had these always be generated for all deployments and merged in local values. Do we see any value in just doing that here, since presumably we need to inject these config/credential values for all deployments anyway, not just loical.

Reply:
Presumably these files need to exist, does anything make sure of that, or does the deploy just fail and we have it in the readme or soemthing?